plugins/woocommerce/src/Admin/PageController.php

```diff
@@ -440,7 +440,7 @@ public function register_page( $options ) {
 			'capability' => 'view_woocommerce_reports',
 			'path'       => '',
 			'icon'       => '',
-			'position'   => null,
+			'position'   => 0,
```

Comment:
Hey @defive Thanks for jumping to fix it! I think a better way may be to still keep the default value as `null` but skip to do the `round()` and `intval()` if the position is `null`. What do you think?

Reply:
Hi @chihsuan sure, that sounds like a good idea. I'll create another PR with that change instead